poolrpc/trader.proto

```diff
@@ -1011,6 +1018,35 @@ message RecoverAccountsResponse {
     uint32 num_recovered_accounts = 1;
 }
 
+message AccountModificationFeesRequest {
+}
+
+message AccountModificationFee {
+    // Modification action type.
+    string action = 1;
+
+    // Action transaction block height.
+    int32 block_height = 2;
+
+    // Action transaction timestamp.
+    int64 timestamp = 3;
+
+    // Action transaction output amount.
+    int64 output_amount = 4;
+
+    // Action transaction fee.
+    int64 fee = 5;
```

Comment:
I've implemented a `oneof` field in my latest commit:
```
    // Action transaction fee.
    oneof fee {
        // A flag which is true if fee value has not been set, and is otherwise
        // false.
        bool fee_null = 5;

        // Action transaction fee value.
        int64 fee_value = 6;
    }
```

where `fee_null` is a flag that should be considered before reading `fee_value`.

Let me know what you think!

Reply:
Just seeing this now. The `fee_null` field works for me 👍